### ISSUE TITLE
\item {\bf } creating wrong&invalid output

### ISSUE THREAD

**Issue by pkra** [AmerMathSoc/texml#224]
The following TeX

```tex
\begin{enumerate}
    \item {\bf Prove:} Let...
\end{enumerate}    
```

generates 

```xml
    <def-list content-type="enumerate">
      <bold>
        <def-item id="ltxid3"><term>(1)</term> <def><p>Prove: Let...</p></def></def-item>
      </bold>
    </def-list>
```

Note how the bold tag wraps around the entire def-item.

This looks like a recent regression (since this example is from a publication where the old XML has the bold in the correct position).

**Comment by davidmjones** [comment]
@pkra This should be fixed now.  Can you confirm?

**Comment by pkra** [comment]
@davidmjones I can confirm. 

I'm guessing this was resolved by 933e49ac12ec73c9130b5a2fc0f66030690b07f3
